src/server.rs

```diff
@@ -380,6 +380,10 @@ async fn index(
     req: HttpRequest,
 ) -> impl Responder {
     let mut request = Request::from_actix_request(&req, body, &global_request_headers);
```

Comment:
Should the ip_addr not be set in this function?

Reply:
You're right! Was kinda planning on doing it and just missed it in the end, will move it.